kubernetes/utils/delete_from_yaml.py

```diff
@@ -0,0 +1,146 @@
+import re
```

Comment:
please add copyright boilerplate

Reply:
sure Ill add the boilerplate and try writing the tests

Reply:
added the boilerplate and added tests also modifying delete_from_dict function